.changeset/add-new-client-method-to-elastic-search-egine.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-search-backend-module-elasticsearch': minor
```

Comment:
Just a note: while we're on `0.x.y` releases, API additions are treated as `patch` and only breaking changes are being treated as `minor`.

Another search contributor pro tip: if you prefix the name of this changeset with `search-`, it narrows the scope of the required reviewers a bit.